src/core/linear_solver/src/preconditioner/4C_linear_solver_preconditioner_ifpack.cpp

```diff
@@ -26,11 +26,11 @@ Core::LinearSolver::IFPACKPreconditioner::IFPACKPreconditioner(
 
 //------------------------------------------------------------------------------
 //------------------------------------------------------------------------------
-void Core::LinearSolver::IFPACKPreconditioner::setup(Epetra_Operator* matrix,
+void Core::LinearSolver::IFPACKPreconditioner::setup(Core::LinAlg::SparseOperator* matrix,
```

Comment:
Since you touch the call-sites anyway, how about making this a reference? The first line in this function shows that this pointer must be non-null.